src/modules/map/components/openlayers/OpenLayersMap.vue

```diff
@@ -96,15 +96,20 @@
         />
     </div>
     <!-- So that external modules can have access to the map instance through the provided 'getMap' -->
+    <slot name="openlayers" />
```

Comment:
as said in `src/modules/map/MapModule.vue` I would prefer not having this `slot` / `template` approach to switch between renderer, but instead having the `MapModule` load on or the other based on `state.ui.showIn3D`